source/admin_manual/list_of_events.rst

```diff
@@ -2333,6 +2333,38 @@ happens.
 | object_id             | Object ID in the storage                             |
 +-----------------------+------------------------------------------------------+
 
+fs_dictmap_max_bucket_changed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.3.13
+
+This event is sent whenever the ``max_bucket`` value for a mailbox changes.
+There can be two situations this happens. Either a new mail is added to a
+mailbox, where the current bucket is found to be filled and a next bucket is
+started to get filled (``reason == file``).
+
+Besides the latter, expected situation, Dovecot emits this event if it
+encounters a bucket with a higher index then the current max_bucket while
+iterating a mailbox (``reason == iter``).
+
++-----------------------+------------------------------------------------------+
+| Field                 | Description                                          |
++=======================+======================================================+
+| success               | Updating ``max_bucket`` in dict & fs-bucket-cache was|
+|                       | successful: ``yes`` or ``no``.                       |
++-----------------------+------------------------------------------------------+
+| msg                   | Error message in case success was ``no`` otherwise   |
+|                       | explanatory message, changing with the reason field. |
```

Comment:
I'd split this into two fields: "error" and "reason". And if there's "error" it makes the "success" field redundant so it could be removed.

Except you have a "reason" already below .. what is the reason message here? Is it just for human-readability (which shouldn't be necessary in events) or does it provide some extra information that reason=file|iter alone doesn't? In that case maybe add more machine-readable fields rather than human-readable.

Reply:
I added the `msg` to be the same as the warning/debug message that would have been logged. So it was clearly human-readable and is not needed in this event.

I changed it now to only have an "error" field. The "reason" field gives all other needed information, thanks for pointing out.